genrl/agents/deep/base/offpolicy.py

```diff
@@ -174,7 +174,7 @@ def select_action(
 
         # add noise to output from policy network
         if self.noise is not None:
-            action += self.noise()
+            action = action + self.noise()
```

Comment:
Is there a reason for this?

Reply:
It works fine without this change as well. My Bad